pkgs/development/python-modules/yamlpath/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchPypi
+, isPy3k
+, ruamel-yaml
+}:
+let
+  pinned-ruamel-yaml = ruamel-yaml.overridePythonAttrs (
+    old: rec {
+      pname = "ruamel.yaml";
+      version = "0.17.10";
+      src = fetchPypi {
+        inherit pname version;
+        sha256 = "EGvI1txqD/fJGWpHVwQyA29B1Va3eca05hgIX1fjnmc=";
+      };
+    }
+  );
+in
+buildPythonPackage rec {
+  pname = "yamlpath";
+  version = "3.6.3";
+
+  src = fetchFromGitHub {
+    owner = "wwkimball";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "4lLKMMsjVWbnfiaOzdBePOtOwPN8nui3Ux6e55YdGoo=";
+  };
+
+  disabled = !isPy3k;
+
+  propagatedBuildInputs = [ pinned-ruamel-yaml ];
+
+  # Tests need more dependencies that are not in nixpkgs
```

Comment:
Please list them.

Reply:
I found all of them now and got the tests working :)